README.md

```diff
@@ -72,34 +53,33 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 
-[android]: https://www.android.com/
 
-[snowplow]: https://snowplowanalytics.com
-[snowplow-java-tracker]: https://github.com/snowplow/snowplow-java-tracker
+[website]: https://snowplowanalytics.com
+[snowplow]: https://github.com/snowplow/snowplow
+[docs]: https://docs.snowplowanalytics.com/
+[mobile-docs]: https://docs.snowplowanalytics.com/docs/collecting-data/collecting-from-own-applications/mobile-trackers/
 
-[micro]: https://github.com/snowplow-incubator/snowplow-micro
-
-[techdocs-image]: https://d3i6fms1cm1j0i.cloudfront.net/github/images/techdocs.png
-[quick-start-image]: https://d3i6fms1cm1j0i.cloudfront.net/github/images/setup.png
-[roadmap-image]: https://d3i6fms1cm1j0i.cloudfront.net/github/images/roadmap.png
-[contributing-image]: https://d3i6fms1cm1j0i.cloudfront.net/github/images/contributing.png
+[gh-actions]: https://github.com/snowplow/snowplow-objc-tracker/actions
+[gh-actions-image]: https://github.com/snowplow/snowplow-android-tracker/workflows/Build/badge.svg
 
-[techdocs]: https://docs.snowplowanalytics.com/docs/collecting-data/collecting-from-own-applications/android-tracker/android-1-4-0/
-[techdocs-old]: https://github.com/snowplow/snowplow/wiki/Android-Tracker
-[setup]: https://docs.snowplowanalytics.com/docs/collecting-data/collecting-from-own-applications/android-tracker/android-1-4-0/#quick-start
-[roadmap]: https://github.com/snowplow/snowplow/wiki/Product-roadmap
+[coveralls]: https://coveralls.io/github/snowplow/snowplow-android-tracker?branch=master
+[coveralls-image]: https://coveralls.io/repos/github/snowplow/snowplow-android-tracker/badge.svg?branch=master
 
-[gh-actions]: https://github.com/snowplow/snowplow-android-tracker/actions
-[gh-actions-image]: https://github.com/snowplow/snowplow-android-tracker/workflows/build/badge.svg?branch=master
+[license]: https://www.apache.org/licenses/LICENSE-2.0
+[license-image]: https://img.shields.io/cocoapods/l/SnowplowTracker.svg
```

Comment:
I've just spotted this sorry! Looks like this is using the Cocoapods license shield (which is a bit odd on Android), I think for consistency with other Snowplow repos, we should use the GitHub license: https://img.shields.io/github/license/snowplow/snowplow-android-tracker

Reply:
Ah damn, I've copied from the iOS repo one and I didn't noticed that one. Guess it's wrong there too. I'll use the Prepare for release commit to fix it on iOS. Also, I just noticed coveralls badge no longer work. Hope it's a momentary issue in their side.

Reply:
Yeah I noticed the coveralls one, I assume its temporary. If it doesn't resolve itself we can fix it/remove it in v2.0.1 - it doesn't seem to work on Shields.io either https://img.shields.io/coveralls/github/snowplow/snowplow-android-tracker